src/models/session/logic.test.ts

```diff
@@ -122,8 +122,8 @@ describe("getPathForNode", () => {
   it("returns a path for a complex flow", () => {
     const path = getPathForNode({ nodeId: "kTEuqpqCh2", flow });
 
-    expect(path).toHaveLength(58);
-    expect(path[57].id).toBe("_root");
+    expect(path).toHaveLength(62);
+    expect(path[61].id).toBe("_root"); // _root is always last
```

Comment:
The mock flows changed here so the length of node paths has legitimately changed too